cmd/sointu-track/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 
 	trackerUi := gioui.NewTracker(model)
 	audioCloser := audioContext.Play(func(buf sointu.AudioBuffer) error {
-		player.Process(buf, midiContext, trackerUi)
+		player.Process(buf, midiContext)
```

Comment:
this third argument was a relic from my old solution, which is why you introduced the broker. ergo, I got rid of it